include/ttmlir/Dialect/TTIR/Transforms/Passes.td

```diff
@@ -87,7 +87,7 @@ def TTIRLoadSystemDesc: Pass<"ttir-load-system-desc", "::mlir::ModuleOp"> {
   }];
 
   list<Option> options = [
-        Option<"path", "path", "std::string", "", "System desc path">,
+        Option<"systemDescPath", "system-desc-path", "std::string", "", "System desc path">,
```

Comment:
I think this is going to break a lot of folks scripts, I also think it's a bit redundant since it's a sub-option of `load-system-desc`, I'd prefer to just keep `path`.

Reply:
sure!

Reply:
Wait, I thought we decided not to change this one

Reply:
alright talked to nick offline, this is the final plan
pass = path, pipeline = system-desc-path